modules/vaos/spec/request/preferences_request_spec.rb

```diff
@@ -36,5 +36,28 @@
         end
       end
     end
+
+    context 'with a valid PUT preferences request', :skip_mvi do
+      let(:user) { build(:user, :vaos) }
+      let(:request_body) do
+        {
+          notification_frequency: 'Each new message',
+          email_allowed: true,
+          email_address: 'abraham.lincoln@va.gov',
+          text_msg_allowed: false,
+          text_msg_ph_number: ''
+        }
+      end
+
+      it 'returns a 200 with correct schema' do
+        VCR.use_cassette('vaos/preferences/put_preferences', match_requests_on: %i[method uri]) do
+          put '/v0/vaos/preferences', params: request_body
+
+          expect(response).to have_http_status(:ok)
+          expect(response.body).to be_a(String)
+          expect(response).to match_response_schema('vaos/preferences')
+        end
+      end
+    end
```

Comment:
Looks like a happy path test, but can we test the sad-paths?

Reply:
We have certain degree of confidence that our middleware are handling the really unhappy paths like 500 already, we certainly wouldn't write specs for those. And I couldn't get any of the attributes here to return errors. I'd have to sit and reverse engineer each one until I find something that returns an error. Would prefer to wait for FE to notify me that something's causing issues and addressing at that point than to spend extra time on reverse engineering unhappy paths.

Reply:
I understand this approach - just wanted to make sure any *documented* paths or known errors are captured.